blokus_env/blokus_env.py

```diff
@@ -96,17 +117,20 @@ def step(self, action):
                     horizontal_flip=horizontal_flip, vertical_flip=vertical_flip
                 )                
                 reward = -5  # Invalid move penalty
+                self.scores[self.current_player - 1] += len(PIECES[list(PIECES.keys())[piece_index]])
             else:
                 piece_name = list(PIECES.keys())[piece_index]
                 reward = len(PIECES[piece_name])
+
+                self.scores[self.current_player - 1] += reward*2
```

Comment:
If you are measuring game score, I think you should only count how many pieces the player has been able to put in the board (no matter if the piece placement action is intentional or not). Hence, I would just use:

`self.scores[self.current_player - 1] += len(PIECES[list(PIECES.keys())[piece_index]])`

after any move

Reply:
Yes, it does seem more appropriate. I'll change that! And in some configs, the players are making more random placements than intentional so their scores are negative...